src/core/Menu/Menu.tsx

```diff
@@ -17,10 +18,21 @@ export type MenuProps = {
   role?: string;
   /**
    * Menu items. Recommended to use `MenuItem` components.
+   *
+   * If you have custom actionable items, they should have `tabindex={-1}`
+   * because we are using rowing `tabindex` for keyboard navigation: https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex.
    */
   children: React.ReactNode;
 } & Omit<CommonProps, 'title'>;
 
+const getFocusableItems = (container: HTMLElement | null) => {
+  return [
+    ...getFocusableElements(container),
```

Comment:
Why do you have to do this instead of just looking through the children to find the elements?

Reply:
I also wanted to handle complex cases like this:
![image](https://user-images.githubusercontent.com/36186912/134372213-0f967958-2f37-47b4-b11a-45c0bfd88003.png)